arch/arm/src/nrf52/Kconfig

```diff
@@ -14,25 +14,31 @@ config ARCH_CHIP_NRF52832
 	bool "NRF52832"
 	select ARCH_FAMILY_NRF52
 	select ARCH_FAMILY_NRF52832
```

Comment:
This is what I mentioned in my comment for the PR. ARCH_FAMILY_NRF52832 is redundant with ARCH_CHIP_NRF52832. If there are things in common with 32 and 33 it should be named more general like NRF52283x.

Reply:
DONE